ale/drivers/base.py

```diff
@@ -94,6 +97,60 @@ def to_pfeffer_response(self):
         data['name_sensor'] = data['instrument_id']
         return data
 
+    @property
+    def t0_ephemeris(self):
+        return self.starting_ephemeris_time - self.center_ephemeris_time
+
+    @property
+    def t0_quaternion(self):
+        return self.starting_ephemeris_time - self.center_ephemeris_time
+
+    @property
+    def dt_ephemeris(self):
+        return (self.ending_ephemeris_time - self.starting_ephemeris_time) / self.number_of_ephemerides
+
+    @property
+    def dt_quaternion(self):
+        return (self.ending_ephemeris_time - self.starting_ephemeris_time) / self.number_of_ephemerides
+
+class TransverseDistortion(ABC):
+    """
+    Exposes the properties that are used to describe a transverse distortion model.
+    """
+    @property
+    @abstractmethod
+    def odtx(self):
+        pass
+
+    @property
+    @abstractmethod
+    def odty(self):
+        pass
+
+
+class RadialDistortion(ABC):
+    """
+    Exposes the properties that are used to describe a radial distortion model.
+    """
+    @property
+    @abstractmethod
+    def odtk(self):
+        pass
+
+
+class LineScanner(Base):
+    @property
+    def name_model(self):
+        return "USGS_ASTRO_LINE_SCANNER_SENSOR_MODEL"
+
+
+class Framer(Base):
+    @property
+    def name_model(self):
+        return "USGS_ASTRO_FRAME_SENSOR_MODEL"
+
+
+class PDS3():
     def _compute_ephemerides(self):
```

Comment:
designed as a mixin?

Reply:
yeah, by itself meaningless but can be used for any instrument driver using PDS3 Labels.